### ISSUE TITLE
PCSX-Redux shows me a black screen when trying to buy and watch the Scylla tactic in Castlevania Symphony of the Night whose game ID is SLUS00067

### ISSUE THREAD

**Issue by buruto** [grumpycoders/pcsx-redux#1139]
I am using the last version of PCSX-Redux which can open, boot and play the game for me that is 11860.20221203.9.x64-cli.

The Scylla tactic costs 1200 gold. With Alucard I have much more than 1200 gold so I can buy the tactic and watch it but when I do PCSX-Redux shows me a black screen and I must load state to escape this but I can't watch the tactic through PCSX-Redux.

Either I must watch it on another playstation emulator like DuckStation or YouTube.

The three other tactics before Scylla I can buy and watch without any problem:

1. Dracula
2. Slogra&Gaibon
3. Doppleganger10

Maybe this is something not right that I did but I am not sure.

Ask me any questions if I forgot to or didn't provide some important details or if this is necessary.

**Comment by nicolasnoble** [comment]
Your iso is broken, as per #1137.

Things work here properly. Use the Debug->Show Iso Browser widget to compute the CRC32 of your iso and verify against redump: http://redump.org/disc/3379/

![image](https://user-images.githubusercontent.com/7281574/206635023-0fc8b958-1490-44c5-992b-607a32b04a64.png)

![image](https://user-images.githubusercontent.com/7281574/206635004-7700f8b0-15b0-4ab3-bd6e-d6702ef19728.png)

**Comment by buruto** [comment]
> Your iso is broken, as per #1137.
> 
> Things work here properly. Use the Debug->Show Iso Browser widget to compute the CRC32 of your iso and verify against redump: http://redump.org/disc/3379/
> 
> ![image](https://user-images.githubusercontent.com/7281574/206635023-0fc8b958-1490-44c5-992b-607a32b04a64.png)
> 
> ![image](https://user-images.githubusercontent.com/7281574/206635004-7700f8b0-15b0-4ab3-bd6e-d6702ef19728.png)

Perhaps I broke the functionality of this game with all the cheats that I made with the memory editor of PCSX-Redux.

I will save my progress to a memory card, then hard reset PCSX-Redux and then load my progress from the same memory card where I saved my progress at first and then let's see if the image itself is broken or not.

**Comment by buruto** [comment]
I found it!

With Memory Editor #1 of PCSX-Redux I poked the following memory address: 800FD8E0

The original instruction there is 14620034 (34006214 in little endian) whose assembly is `bne $v1, $v0, 0x800fd9b4` as how it is displayed in the Assembly window or tab of PCSX-Redux and I replaced this word with 00000000 that is `nop`.

I found this action necessary to get 99 instances of the purchased item in a single buy. I didn't want to buy the same item 99 times because this is taking too much time so I just made this patch.

Don't worry about gold. I made patches that make all the items free in the shop which were gone after the left shop and before I returned to the shop though.

A pity that the problematic patch was not gone in this period.

So all what I had to do is to use the Memory Editor of PCSX-Redux to bring back the original instruction by just poking again 800FD8E0 and replace 00000000 by 34006214 (`nop` => `bne $v1, $v0, 0x800fd9b4`).

This was my mistake and I shouldn't have open this issue in first place.

I mustn't open issues that I created with the memory editor of PCSX-Redux and I won't do or repeat this mistake again in the future.
